data/scripts/spells/monster/gaz'haragoth_summon.lua

```diff
@@ -1,31 +1,44 @@
+dofile("data/scripts/spells/monster/gaz_functions.lua")
+
 local spell = Spell("instant")
 
 function spell.onCastSpell(creature, var)
-local t, spectator = Game.getSpectators(creature:getPosition(), false, false, 5, 5, 5, 5)
+    local t, spectator = Game.getSpectators(creature:getPosition(), false, false, 25, 25, 25, 25)
```

Comment:
value assigned to variable 'spectator' is unused

Reply:
It is being used, for some reason the check is not identifying.